pvlib/pvsystem.py

```diff
@@ -511,6 +511,22 @@ def sapm_effective_irradiance(self, poa_direct, poa_diffuse,
             poa_direct, poa_diffuse, airmass_absolute, aoi,
             self.module_parameters, reference_irradiance=reference_irradiance)
 
+    def pvsyst_celltemp(self, poa_global, wind_speed, temp_air, eta_m=0.1,
+                        alpha_absorption=0.9, temp_model="freestanding"):
```

Comment:
I believe the default kwarg `"freestanding"` is redundant because it will be overwritten by the `racking_model='open_rack_cell_glassback'` argument passed for `temp_model` on L528 below

Reply:
We need to reconcile the two sets of keywords, one for the SAPM temperature model and the other for the PVsyst  model.  A topic for a separate PR, I think.